Shared/mods/deathmatch/logic/CLodModels.cpp

```diff
@@ -0,0 +1,4355 @@
+    {13722, 13759},            // vinesign1_cunte => lodvinesign1_cunte (LAhills)
+    {13723, 13819},            // cunte_hollyhil01 => lodcunte_hollyhil01 (LAhills)
+    {13724, 13783},            // drg_nu_ext => loddrg_nu_ext (LAhills)
+    {13725, 13855},            // opmans01_cunte => lodopmans01_cunte (LAhills)
+    {13726, 13786},            // road_hill08 => lodroad_hill08 (LAhills)
+    {13727, 13760},            // cenorthbrij01 => lodcenorthbrij01 (LAhills)
+    {13729, 13889},            // cegravebuil01 => lodravebuil02 (LAhills)
+    {13730, 13844},            // ceroad_6 => lodceroad_6 (LAhills)
+    {13732, 13770},            // ce_roads37 => lodce_roads37 (LAhills)
+    {13733, 13780},            // ce_roads42 => lodce_roads42 (LAhills)
+    {13734, 13768},            // hillclif06 => lodhillclif06 (LAhills)
+    {13735, 13779},            // ce_roads41 => lodce_roads41 (LAhills)
+    {13736, 13778},            // ce_roads40 => lodce_roads40 (LAhills)
+    {13737, 13767},            // cuntehill03 => lodcuntehill03 (LAhills)
+    {13738, 13771},            // roads39_ce => lodroads39_ce (LAhills)
+    {13739, 13772},            // roads38ce => lodroads38ce (LAhills)
+    {13740, 13750},            // cntehillclif01 => lodtellclif01 (LAhills)
+    {13741, 13765},            // hillclif02 => lodhillclif02 (LAhills)
+    {13742, 13764},            // hillclif05 => lodhillclif05 (LAhills)
+    {13746, 13790},            // tcehillhouse07 => lodtcehillhouse07 (LAhills)
+    {13747, 13791},            // cehillhse06 => lodcehillhse06 (LAhills)
+    {13748, 13878},            // tcemulhilhed1_law01 => lod_ilhed1_law01 (LAhills)
+    {13749, 13860},            // cunte_curvesteps1 => lodcunteteps1 (LAhills)
+    {13751, 13781},            // cunte_flyover2 => lodcunte_flyover2 (LAhills)
+    {13752, 13876},            // cuntebridge01 => lod_cuntebridge01 (LAhills)
+    {13753, 13799},            // cehillhouse04 => lodcehillhouse04 (LAhills)
+    {13755, 13794},            // cehillhouse01 => lodcehillhouse01 (LAhills)
+    {13756, 13879},            // hollyhil04a => lod_hollyhil04a (LAhills)
+    {13757, 13881},            // hollyhil05 => lod_hollyhil05 (LAhills)
+    {13761, 13762},            // cunte_whisky => lodcunte_whisky (LAhills)
+    {13784, 13836},            // road_hill01 => lodroad_hill01 (LAhills)
+    {13789, 13851},            // roads40_ce => lodroads40_ce (LAhills)
+    {13795, 13796},            // ce_hillseast06 => lodtehillseast06 (LAhills)
+    {13801, 13880},            // cunte_skatetrak => lod_cunte_skatetrak (LAhills)
+    {13804, 13808},            // cuntelandf4 => lodcuntelandf4 (LAhills)
+    {13805, 13807},            // celalandbiv => lodcelalandbiv (LAhills)
+    {13809, 13811},            // ce_grndpalcst02 => lodcepalcst02 (LAhills)
+    {13810, 13812},            // ce_grndpalcst05 => lodce_grndpalcst05 (LAhills)
+    {13813, 13815},            // 
```

Comment:
You should reset the list before connecting to a new server

Reply:
I tested, it is resetted when changing server.